### ISSUE TITLE
[TOPIC] How to configure BigQuery Partition, Clustering Keys, etc with Custom Transformations

### ISSUE THREAD

**Issue by ChristopheDuong** [airbytehq/write-for-airbyte#53]
**Request Details**
What type tutorial/article would you like to see?

dbt allows a long list of configurations of materialization options:
https://docs.getdbt.com/reference/resource-configs/bigquery-configs

If a user exports the generated normalization project from a sync and re-import it as a custom operation, they should be able to edit and customize advanced configurations such as these (not only applicable to bigquery but to other destinations too)

Examples:
- Using table partitioning and clustering
- (in other databases, set primary keys?)
- Policy tags: BigQuery enables column-level security by setting policy tags on specific columns.

**Comment by tuanchris** [comment]
Hi @avaidyanatha, @ChristopheDuong this sounds like a good article I could write. Can I claim this & how do I get started?

**Comment by avaidyanatha** [comment]
@tuanchris Absolutely, it's yours. We should get on a call to make sure that we are aligned on the direction of the piece.

**Comment by arimbr** [comment]
Draft: https://docs.google.com/document/d/1HFdKro6qgN-k2_t_gJzWdUsWl4ZVP1z-Fl8VX66uOw8/edit?usp=sharing
